src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Avalonia.LinuxFramebuffer;
+
+internal static class EnumerableExtesions
+{
+    public static IEnumerable<TSource> Where<TSource, TArg>(this IEnumerable<TSource> source, Func<TSource, TArg, bool> predicate, TArg arg)
```

Comment:
Do we really need this extension here? Simple one pass foreach can work there just fine.

Reply:
Because I would like to avoid closure allocation

Reply:
Isn't this code executed once? Either way, with foreach the same closure can be avoided as well.

Reply:
Note, in Linq Where operator has number of optimizations depending on the input collection type. https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/src/System/Linq/Where.cs#L36C28-L36C46